src/netcdf_scm/masks/__init__.py

```diff
@@ -111,6 +112,7 @@ def f(masker, cube, **kwargs):
     return f
 
 
+@lru_cache(maxsize=32)
```

Comment:
You would want to check if this is a per instance cache or if all instances use the same cache. If that is the case it doesn't do anything unless `get_default_sftlf_cube` is called multiple times per instance.

Reply:
> If that is the case it doesn't do anything unless get_default_sftlf_cube is called multiple times per instance

It'll get called multiple times per instance so I think good enough?

Reply:
I was mistaken it is a module level function not a class member (Why does GH's first indent have to be so big).